### ISSUE TITLE
Scala not working out of box

### ISSUE THREAD

**Issue by cliebhardt** [webforj/webforj#695]
When trying to use webforj with Scala the following error message is displayed  on the scala:compile goal

```
[ERROR] : Bad symbolic reference. A signature
refers to BBjWindow/T in package com.basis.bbj.proxies.sysgui which is not available.
It may be completely missing from the current classpath, or the version on
the classpath might be incompatible with the version used when compiling the signature.
```

To fix the Issue the following commands have to be run in bbx/.libs:

```
mvn install:install-file "-Dfile=BBjStartup.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjStartup" "-Dversion=24.10" "-Dpackaging=jar"
mvn install:install-file "-Dfile=BBj.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBj" "-Dversion=24.10" "-Dpackaging=jar"
mvn install:install-file "-Dfile=BBjUtil.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjUtil" "-Dversion=24.10" "-Dpackaging=jar"
mvn install:install-file "-Dfile=BBjsp.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjsp" "-Dversion=24.10" "-Dpackaging=jar"
mvn install:install-file "-Dfile=BBjAdminAPI.jar" "-DgroupId=com.basis.lib" "-DartifactId=BBjAdminAPI" "-Dversion=24.10" "-Dpackaging=jar"
```

**Note**: Use the corresponding version to your webforj version. On Linux distrobutions the commands might need be executed with root permissions.

Afterwards added the following dependencies to your pom.xml:

```xml
<dependency>
  <groupId>com.basis.lib</groupId>
  <artifactId>BBj</artifactId>
  <version>24.10</version>
  <scope>provided</scope>
</dependency>
<dependency>
  <groupId>com.basis.lib</groupId>
  <artifactId>BBjStartup</artifactId>
  <version>24.10</version>
  <scope>provided</scope>
</dependency>
<dependency>
  <groupId>com.basis.lib</groupId>
  <artifactId>BBjUtil</artifactId>
  <version>24.10</version>
  <scope>provided</scope>
</dependency>
```

This steps should fix the issue and make it possible to use webforj with Scala.

**Software Versions:**
 - Java Version: 17
 -  Scala Version: 3.3.3
 - BBj Version: 24.10

**Desktop:**
 - OS: Ubuntu
 - Browser Firefox
 - Version 128.0.3

**Comment by hyyan** [comment]
@StephanWald 

related to #562

**Comment by hyyan** [comment]
Fixed in 24.20
